portfolio/src/main/webapp/index.html

```diff
@@ -8,8 +8,22 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
+      <div class="h1-container">
```

Comment:
I don't see any CSS for `.h1-container`, do you still need this?  Oftentimes you can style the h1 itself instead of wrapping it in a div